### PR TITLE
New recipe: Peggy

### PR DESCRIPTION
**Pull request by andimachovec**
Peggy is a mastermind clone for Haiku written in C++ using the native Haiku API

### CODE REVIEW

File: haiku-games/peggy/peggy-0.5.2.recipe [hunk 1]
```diff
@@ -0,0 +1,43 @@
+SUMMARY="A board game inspired by Mastermind(tm)"
+DESCRIPTION="In this game you have to crack a color code in 9 attempts\
```
Comment (humdingerb) [inline haiku-games/peggy/peggy-0.5.2.recipe:2]: Needs a space before the \\-escaped newline or you'll see "...attemptsby getting...".

File: haiku-games/peggy/peggy-0.5.2.recipe [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+by getting feedback about the correctness of the color pegs and their positions"
```
Comment (humdingerb) [inline haiku-games/peggy/peggy-0.5.2.recipe:3]: Add a full-stop.

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: haiku-games/peggy/peggy-0.5.2.recipe (hunk 1 above)
Comment (andimachovec) [reply haiku-games/peggy/peggy-0.5.2.recipe]: Done. Thanks humdinger! :-)

File: haiku-games/peggy/peggy-0.5.2.recipe (hunk 2 above)
Comment (andimachovec) [reply haiku-games/peggy/peggy-0.5.2.recipe]: Done as well.

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**